src/uct/ib/dc/dc_mlx5_devx.c

```diff
@@ -50,7 +52,13 @@ ucs_status_t uct_dc_mlx5_iface_devx_create_dct(uct_dc_mlx5_iface_t *iface)
     if (!uct_ib_iface_is_roce(&iface->super.super.super)) {
         UCT_IB_MLX5DV_SET(dctc, dctc, pkey_index, ib_iface->pkey_index);
     }
-    UCT_IB_MLX5DV_SET(dctc, dctc, port, ib_iface->config.port_num);
+
+    if((dev->lag_level < 2) ||
```

Comment:
do we really need the first check?

Reply:
@brminich 
Technically, in https://github.com/openucx/ucx/blob/v1.14.0-rc4/src/uct/ib/mlx5/dv/ib_mlx5dv_md.c#L1049-L1056
1) ```lag_level``` could be 0 if it failed to query lag state whatever it's roce lag or not.
2) ```lag_level``` could be 1 if it's not in lag state
3) ```lag_level```  could be the queried out value.
Considering the [ucs_rand_range](https://github.com/openucx/ucx/blob/v1.14.0-rc4/src/ucs/sys/math.c#L48-L57) implementation, it do the ```first``` check here.

Could you help mark the label ```WIP-DNM``` for this PR? I added ```[DNM]``` in this PR's description. As I said in the **Why?** section, this PR is required by the manager, I'm on the way to verify it on CX5.

Reply:
ok, i did not see that it could be `0`. seems that `1` would not be a problem

Set `DNM` label